client/core/trade.go

```diff
@@ -704,6 +704,13 @@ func (t *trackedTrade) redeemMatches(matches []*matchTracker) error {
 			match.failErr = err
 			errs.add("error sending 'redeem' message for match %s, coin %x: %v",
 				match.id, coinIDString(redeemAsset.ID, coinID), err)
+			// TODO: Do not skip this match, set the status to MakerRedeemed.
+			// Redeem tx was broadcasted, and tests have shown cases where the
+			// server got the redeem request but the client did not receieve
+			// the server's ack. In such cases, the counter-party is notified
+			// of this redeem and the trade proceeds as it should. The described
+			// scenarios usually have err = "timed out waiting for 'redeem'
+			// response."
 			continue
```

Comment:
Running the simnet trading tests repeatedly should produce a failed `TestTrading/success` test every once in a while because one client did not receive server ack for a redeem request. The test logs would have errors like:
```
[ERR] CORE: notify: |ERROR| (order) Redemption error - Error encountered sending redemptions worth 0.00180000 btc on order 98099718 - Order: 980997182e82d1984f4de5d7e38cd6aa6f8c311847bb288d5a61703abc01c5bc
[ERR] CORE: 127.0.0.1:17273 tick: redeemMatches: redeemMatches - order 980997182e82d1984f4de5d7e38cd6aa6f8c311847bb288d5a61703abc01c5bc - error sending 'redeem' message for match 808b70bcebf47b138fccd8cf1eebf126df4340e54ae7f56712e6c72dd75d1d97, coin 663536323430623464383061386463376161323466653534323239383635653832333730386631633838346661623164313536656234316130643637396235373a30: timed out waiting for 'redeem' response.
```

Reply:
Handled in #459.